### ISSUE TITLE
Add stack trace to .attrs warning

### ISSUE THREAD

**Issue by mxstbr** [styled-components/styled-components#2481]
We should log the stack trace when warning about function keys in `.attrs`. Ref: https://spectrum.chat/styled-components/help/functions-as-object-form-attrs-error-but-not-in-source~390a2294-3b80-4527-a2ea-36556a540ed8

**Comment by lewisflude** [comment]
Just saw the thread on spectrum!

I believe I may be running into the same problem. Just wanted to post the following code snippet that's triggering the warning for me in case it's a false positive.

```
const tooltipFadeIn = keyframes`
    from {
        opacity: 0
    }
    to {
        opacity: 1;
    }
`;

const TooltipWrapper = styled.div.attrs<TooltipWrapperProps>({
  style: ({ xPosition, yPosition, opacity }: TooltipWrapperProps) => ({
    left: xPosition,
    top: yPosition,
    opacity,
  }),
})<TooltipWrapperProps>`
  position: absolute;
  pointer-events: none;
  animation: ${tooltipFadeIn} 80ms linear 1;
`;
```

**Comment by probablyup** [comment]
Closed via #2486
